packages/ui/src/components/Charts/helpers/calculateCountForDataOnDates.ts

```diff
@@ -29,57 +26,40 @@ export function calculateCountForDataOnDates<T extends Record<string, string[]>>
   // Initialize the indices for each data item
   const indices = {} as Record<StringKeys<T>, number>;
   for (const name in data) {
-    indices[name as StringKeys<T>] = 0;
+    indices[name] = 0;
```

Comment:
indices is not being used anymore, safe to remove I guess

Reply:
True 😅

Reply:
[3f326ba](https://github.com/synopsisapp/symbiosis-ui/pull/17/commits/3f326ba8bc621bf9173ff90a24eeb653673adbd6)